apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/search/SearchWebScript1.java

```diff
@@ -47,6 +47,19 @@
           + "## query\n"
           + "Object containing subcomponents that build the requested query.\n"
           + "Info about the Search query syntax can be found here: https://xenitsupport.jira.com/wiki/display/APIX/Search+query+syntax\n"
+          + "### special search terms:\n"
+          + "- type: searches for nodes of that type (for example: \"type\" : \"cm:content\")\n"
+          + "- aspect: searches for nodes with that aspect (for example: \"aspect\" : \"cm:titled\")\n"
+          + "- noderef: searches the node with that noderef (for example: \"noderef\" : \"workspace://SpacesStore/f0d15919-3841-4170-807f-b81d2ebdeb80\")\n"
+          + "- parent: searches the nodes with that parent (for example: \"parent\" : \"workspace://SpacesStore/f0d15919-3841-4170-807f-b81d2ebdeb80\")\n"
+          + "- path: searches the nodes with that path (for example: \"path\" : \"/\")\n"
+          + "- category: searches the nodes with that category (for example: \"category\" : \"workspace://SpacesStore/f0d15919-3841-4170-807f-b81d2ebdeb80\")\n"
+          + "- text: searches the nodes with content containing that text (for example: \"text\" : \"this text\")\n"
+          + "- all: searches the nodes with content containing the value (for example: \"all\" : \"search term\")\n"
```

Comment:
@kpiot123 Documentation for all is not complete. Not only is searched in content but also in name and author
https://github.com/xenit-eu/alfred-api/blob/8699ed6ef671a4c9766c289a1632fe7c91e1ac4f/apix-impl/src/main/java/eu/xenit/apix/alfresco/search/FtsNodeVisitor.java#L110

Reply:
I must have been distracted while I was writing that